html-aam/fragile/area-role.html

```diff
@@ -13,12 +13,13 @@
 
 <map name="areamap">
     <area shape="rect" coords="0,0,15,15" href="#" alt="x" data-testname="el-area" data-expectedrole="link" class="ex">
-    <area shape="rect" coords="15,15,31,31" alt="x" data-testname="el-area-no-href" data-expectedrole="generic" class="ex">
+    <area shape="rect" coords="15,15,31,31" alt="x" data-testname="el-area-no-href" class="ex-generic">
```

Comment:
FYI reviewers, this PR just updates a few older tests that expect a specific role match with the implementation-specific acceptable alternates ["none" | "generic" | ""] as agreed upon in https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/48